frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./Navbar.scss";
+import { Link } from "react-router-dom";
+import * as Dialog from "@radix-ui/react-dialog";
+import NavbarMap from "../../assets/navbar_map.svg";
+import NavbarStations from "../../assets/navbar_stations.svg";
+import NavbarUserPage from "../../assets/navbar_user_page.svg";
+
+function Navbar() {
+  return (
+    <nav className="navbar_container">
+      <ul className="navbar_list">
+        <li className="navbar_element">
+          <Link to="/map" draggable="false">
+            <img
+              className="navbar_img"
+              src={NavbarMap}
+              draggable="false"
+              alt="map logo"
```

Comment:
Your website is in English ?

Reply:
With all that coding in English, dailies in English, .... I guess I turned a little bit "Jean-Claude Vandamme"
It's fixed in the last commit !